core/src/main/java/apoc/create/Create.java

```diff
@@ -196,27 +197,46 @@ public Stream<VirtualPathResult> virtualPath(@Name("labelsN") List<String> label
     @Procedure("apoc.create.clonePathToVirtual")
     @Description("Takes the given path and returns a virtual representation of it.")
     public Stream<PathResult> clonePathToVirtual(@Name("path") Path path) {
-        return Stream.of(createVirtualPath(path));
+        // given that it accepts a single path as a parameter the cache is not necessary
```

Comment:
Also, I wouldn't call it the map a cache, since you use it for tracking the relationships already created and not primarily as a performance optimization. Perhaps we could call it a "relationshipMap" or "createdRelationships"?

Reply:
Changed to `relationshipMap`